package.json

```diff
@@ -34,6 +34,8 @@
 		"onCommand:cosmosDB.createAccount",
 		"onCommand:cosmosDB.removeMongoServer",
 		"onCommand:cosmosDB.createMongoDatabase",
+		"onCommand:cosmosDB.createDocDBDatabase",
+		"onCommand:cosmosDB.createDocDBCollection",
```

Comment:
Currently placed these commands related to similar scenarios (creation of mongo and DocDB entities). 
Should these be made alphabetical?

Reply:
I don't have a strong preference